src/marine/viirs_modis_l3_oc2ioda.py

```diff
@@ -63,12 +69,19 @@ def _read(self):
         # get global attributes
         for v in ('platform', 'instrument', 'processing_version',
                   'time_coverage_start'):
-            AttrData[v] = ncd.getncattr(v)
+            GlobalAttrs[v] = ncd.getncattr(v)
         ncd.close()
 
-        valKey = vName['chlor_a'], self.writer.OvalName()
-        errKey = vName['chlor_a'], self.writer.OerrName()
-        qcKey = vName['chlor_a'], self.writer.OqcName()
+        valKey = vName['chlor_a'], iconv.OvalName()
+        errKey = vName['chlor_a'], iconv.OerrName()
+        qcKey = vName['chlor_a'], iconv.OqcName()
+
+        self.VarAttrs[vName['chlor_a'], iconv.OvalName()]['_FillValue'] = -32767.
+        self.VarAttrs[vName['chlor_a'], iconv.OerrName()]['_FillValue'] = -32767.
+        self.VarAttrs[vName['chlor_a'], iconv.OqcName()]['_FillValue'] = -32767
```

Comment:
Line 81: Not sure if it matters but do we need a dot at the end of the line - just to be consistent with the previous two lines?

Reply:
Thank you so much for reviewing this so quickly. The reason the dot is missing on line 81 is because the data types for obs value and error (float) is different from qc (integer). So if we put a dot at the end, it would get a float where integer is expected and throw an error.

Reply:
aha! that makes a lot sense!